data/README.md

```diff
@@ -0,0 +1,9 @@
+# Simorgh Data Samples
+
+These are a set of data samples which represent possbile input to Simorgh from upstream systems (and ultimately from a CMS).  They cover a number of scenarios, most of which are made up with random Lorem Ipsum or real text.
```

Comment:
For these data samples to be intelligible we need a schema somewhere. At very least in this PR we should mention this and mark it as a TODO: (also add a github issue). We can then update the swaggerhub.io schema we wrote a long time ago.

Reply:
https://github.com/bbc/simorgh/issues/91

Reply:
change `possbile input` to `possible input`. Also, remove the double spacing after `CMS).`.